source/agora/consensus/protocol/Nominator.d

```diff
@@ -1166,6 +1157,11 @@ extern(D):
             else if (this.total_adjusted_fee < other.total_adjusted_fee)
                 return 1;
 
+            if (this.consensus_data.tx_set.length > other.consensus_data.tx_set.length)
+                return -1;
+            else if (this.consensus_data.tx_set.length < other.consensus_data.tx_set.length)
+                return 1;
+
```

Comment:
I'm a bit puzzled. If `this` has more transaction than `other`, then `this < other` ? I'd expect the other way around.

Reply:
Well, we first `sort()` and get `front()`. So our best candidate should be the "smallest"

Reply:
Right, it's `CandidateHolder`